python/ray/tests/test_actor.py

```diff
@@ -1431,6 +1431,22 @@ def ping(self):
     assert ray.get(detached_actor.ping.remote()) == "pong"
 
 
+def test_force_kill(ray_start_regular):
+    @ray.remote
+    class Actor(object):
+        def hang(self):
+            # Never returns.
+            ray.get(ray.ObjectID.from_random())
+
+    actor = Actor.remote()
+    result = actor.hang.remote()
+    ready, _ = ray.wait([result], timeout=0.1)
+    assert len(ready) == 0
+    actor.force_kill()
+    with pytest.raises(ray.exceptions.RayActorError):
+        ray.get(result, timeout=1)
```

Comment:
how fast will `ray.available_resources` be updated?

Reply:
It's the same path as an actor/worker dying, so the raylet should update it very soon.

Reply:
It's the same path as an actor/worker dying - the raylet should update it soon after death.